docs/sdk/crashes/android.md

```diff
@@ -180,13 +234,30 @@ public Iterable<ErrorAttachmentLog> getErrorAttachments(ErrorReport report) {
 	Bitmap bitmap = BitmapFactory.decodeResource(context.getResources(), R.mipmap.ic_launcher);
 	ByteArrayOutputStream stream = new ByteArrayOutputStream();
 	bitmap.compress(Bitmap.CompressFormat.JPEG, 100, stream);
-	byte[] bitMapData = stream.toByteArray();
-	ErrorAttachmentLog binaryLog = ErrorAttachmentLog.attachmentWithBinary(bitMapData, "ic_launcher.jpeg", "image/jpeg");
+	byte[] bitmapData = stream.toByteArray();
+	ErrorAttachmentLog binaryLog = ErrorAttachmentLog.attachmentWithBinary(bitmapData, "ic_launcher.jpeg", "image/jpeg");
 
 	/* Return attachments as list. */
 	return Arrays.asList(textLog, binaryLog);
 }
 ```
+```kotlin
+override fun getErrorAttachments(report: ErrorReport?): MutableIterable<ErrorAttachmentLog> {
+	
+	/* Attach some text. */
+	val textLog = ErrorAttachmentLog.attachmentWithText("This is a text attachment.", "text.txt")
+
+	/* Attach app icon. */
+	val bitmap = BitmapFactory.decodeResource(context.getResources(), R.mipmap.ic_launcher)
```

Comment:
I'm wondering if we want to add a line to get the context so people can just copy-paste this?